### DISCUSSION TITLE
Call icon

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#2404] [Request: Icon] [answered]
For the action of starting a call I am searching for an icon. I am currently _misusing_ the VoiceCapable Icon.

**Comment by TheSisb** [comment]
Related: https://github.com/twilio-labs/paste/discussions/2407

**Comment by SiTaggart** [comment, marked as answer]
Hi @thomaschaaf 

A collection of Call related icons are now available on Figma and NPM https://paste.twilio.design/components/icons
